resources/wikia/modules/spec/imageServing.spec.js

```diff
@@ -21,8 +21,8 @@ describe('Image Serving', function () {
 		expect(typeof cutOptions.yOffset2).toBe('number');
 	});
 
-	using('image sizes', [
```

Comment:
@kvas-damian as we discussed offline data provider plugin (https://github.com/jphpsf/jasmine-data-provider) is no longer working with jasmine 2.0<. The last commit in this project was done more than 2 years ago and as you wrote above it used a hole in jasmine framework which is already fixed. I did search for the alternatives and didn't find any.

It is an open topic but till now this data provider plugin was used only in ```imageServing.spec.js```. I want to continue improving JS unit test infrastructure but before investing more time into this topic I'd like to speak with @kenkouot about progress in Mercury project. The plan is to create our desktop skin using Mercury architecture and currentlyJS unit tests in this project are written in QUnit.